README.md

```diff
@@ -12,5 +12,7 @@ This repository is licensed under CC0 1.0 Universell (CC0 1.0) Public Domain Ded
 
 * [protégé](https://protege.stanford.edu/)
 
-## Info
-Read wiki and CONTRIBUTE.md.
+## Contributing and developping
+Please read the [wiki](https://github.com/OpenEnergyPlatform/ontology/wiki) for information about the ontology and its standards.
```

Comment:
```sugestion
... , best practise principles on content and BFO classification.
```

Reply:
Ok, I think this didn't work like I hoped. The sentence above was intended as a suggestion to add to line 16

Reply:
git only track the changes so you can simply keep the existing sentence and add yours at the end. I think the problem was the `...` :)

Reply:
This PR is already merged, I will edit that in a subsequent PR :)